pkg/util/constants/constant_windows.go

```diff
@@ -0,0 +1,6 @@
+package constants
+
+var (
+	//DefaultDogstatsDLogFile points to the dogstatsd stats log file that will be used if not configured
```

Comment:
```suggestion
	// DefaultDogstatsDLogFile points to the dogstatsd stats log file that will be used if not configured
```
Same for the other constant comments